build.bat

```diff
@@ -4,12 +4,13 @@ mkdir data || goto err
 windres resource.rc -O coff -o resource.res || goto err
 gcc -mwindows main.c wallpaper.c parser.c debug.c window.c resource.res -lmingw32 -lshcore -lSDL2main -lSDL2 -o data\lwp.exe || goto err
 xcopy wallpapers data\wallpapers\ /E /Y || goto err
+TYPE wallpapers\default-fullhd\wallpaper.cfg | FIND /V "" > data\wallpapers\default-fullhd\wallpaper.cfg
 copy uninstall.bat data || goto err
 copy defaultWin.cfg data || goto err
 copy LICENSE data\LICENSE.txt || goto err
 echo.
 echo ---                           Done!                                  ---
-echo --- Remember to put SDL2.dll into 'data' directory before installing ---
+copy SDL2.dll data || echo --- Remember to put SDL2.dll into 'data' directory before installing ---
```

Comment:
Github workflow doesn't work with this line, because `||`  sets `ERRORLEVEL `to `1`. 
Add `if/else` statement instead and it should be fine ;)

Reply:
Thanks, what a blunder on my part :)